src/http/ngx_http_wasm_api.c

```diff
@@ -311,7 +311,20 @@ proxy_get_property(int32_t path_data, int32_t path_size,
                                           res_data, res_size);
     }
 
-    return PROXY_RESULT_OK;
+    ngx_http_request_t *r;
+    must_get_req(r);
+    
+    u_char *lowcase_buf = ngx_palloc(r->pool, path_size);
```

Comment:
Could you follow Nginx's code style? You can take a look at other code. And we need to check the returned res.

Reply:
I'm sorry I made some low mistakes. I've just contacted nginx C development. I'll read the document and improve it.

Reply:
I performed the following code style adjustments follow this document. [https://nginx.org/en/docs/dev/development_guide.html#code_style](https://nginx.org/en/docs/dev/development_guide.html#code_style)

- Move variables that are not predefined in the function header to there
- Adjust the indentation of the function definition area to conform to the specification
- Split lines with a single line length of more than 80 characters according to the specification

Also, I have added a check for when the Nginx variable does not exist, which will return `NOT_FOUND` directly to wasm.

If I still have any missed code style issues, please help point them out and I will fix those immediately.